nucypher/config/characters.py

```diff
@@ -14,7 +14,7 @@
 You should have received a copy of the GNU Affero General Public License
 along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 """
-
+from collections import Iterable
```

Comment:
Not sure where do we use `Iterable` here.

Reply:
Removed.